paperqa/agents/tools.py

```diff
@@ -31,13 +32,38 @@ def make_status(
     )
 
 
+def default_status(state: "EnvironmentState") -> str:
```

Comment:
You'll need a `TYPE_CHECKING` block that imports `EnvironmentState` I think for this to work

Reply:
`EnvironmentState` is in this file, so I think we're good here

Reply:
Yep i was wrong here, sorry for false positive comment